src/classes/client.ts

```diff
@@ -76,21 +78,29 @@ export class AIHordeClient extends Client {
 	}
 
 	async loadHordeStyles() {
-		const source = this.config.data_sources?.styles_source ?? `https://raw.githubusercontent.com/db0/Stable-Horde-Styles/main/styles.json`
+		const source = this.config.data_sources?.styles_source || `https://raw.githubusercontent.com/AI-Horde/Stable-Horde-Styles/main/styles.json`
 		const req = await fetch(source)
 		if(!req.status?.toString().startsWith("2")) throw new Error("Unable to fetch styles");
 		const res = await req.json()
 		this.horde_styles = res
 	}
 
 	async loadHordeStyleCategories() {
-		const source = this.config.data_sources?.style_categories_source ?? `https://raw.githubusercontent.com/db0/Stable-Horde-Styles/main/categories.json`
+		const source = this.config.data_sources?.style_categories_source || `https://raw.githubusercontent.com/AI-Horde/Stable-Horde-Styles/main/categories.json`
```

Comment:
please make sure that the link is actually valid, in this case it is not valid and returns a 404